file_name_in: "/box_test/"
file_name_out: "/box_test/"
file_name_hand_point: "/hand_point/hand_points"